### PR TITLE
Add/a4a overview hosting section

### PR DESCRIPTION
**Pull request by andrii-lysenko**
Resolves https://github.com/Automattic/jetpack-genesis/issues/294

## Proposed Changes

Filling information about Hosting options in the Overview page for A4A. We are keeping these in client for now: pfunGA-Hp-p2#comment-1111

Screenshot 2024-03-21 at 5 28 20 PM

## Testing Instructions

- Spin up the branch locally
- Navigate to http://agencies.localhost:3000/overview
- Check that new section is visible and behaves normally

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (andrii-lysenko) [inline client/a8c-for-agencies/sections/overview/body/hosting/index.tsx:20]: I'm a bit rusty with React so feel free to suggest a better way to implement this.

Comment (yashwin) [reply client/a8c-for-agencies/sections/overview/body/hosting/index.tsx:20]: This looks good 👍

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (yashwin) [inline client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: I think since we have a dedicated page for Pressable, we can redirect to `/marketplace/hosting/pressable` and we can keep this constant in https://github.com/Automattic/wp-calypso/blob/093abd50576b2b29536eac84fbf1a6746744f351/client/a8c-for-agencies/components/sidebar-menu/lib/constants.ts

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (yashwin) [inline client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: Same as above: we can redirect to `/marketplace/hosting/wpcom`

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (yashwin) [inline client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: Can we translate the titles also?

File: client/a8c-for-agencies/components/pressable-logo/index.tsx
Comment (yashwin) [inline client/a8c-for-agencies/components/pressable-logo/index.tsx:1]: I think we can add this here: [client/assets/images](https://github.com/Automattic/wp-calypso/blob/1c5ae12a0d29537e8b186bdb1b28391449b6e320/client/assets/images)

**Review by yashwin — COMMENTED** [review]
It works as expected, and code changes look reasonable. 

[Non-blocker] How about we make the whole header clickable to expand/collapse the card instead of only the icons? 

Screenshot 2024-03-22 at 11 57 14 AM

I have also left some comments. Let me know what you think.

File: client/a8c-for-agencies/components/pressable-logo/index.tsx
Comment (andrii-lysenko) [reply client/a8c-for-agencies/components/pressable-logo/index.tsx:1]: Oh, I thought we don't want to have inter-dependencies with `client`. Or with `assets` it's ok? 
Also, we would then just add `svg` there and we will update `height` and `fill` with styles if needed?

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (andrii-lysenko) [reply client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: Yes, sure. Sorry, just missed these.

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (andrii-lysenko) [reply client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: Hmm, I'm wondering if we should move all links then outside from the components, and have a `constants` folder inside `a8c-for-agencies`, so we show they are shared?

Comment (andrii-lysenko) [reply client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: Also, until I think until that PR merged these links wouldn't work. What do you say if we update these later, when marketplace page is fully functional?

File: client/a8c-for-agencies/components/pressable-logo/index.tsx
Comment (yashwin) [reply client/a8c-for-agencies/components/pressable-logo/index.tsx:1]: Yes, I think `client/assets` is ok. 

> Also, we would then just add svg there and we will update height and fill with styles if needed?

Yes, we could do that & keep the default values for what we currently have 👍

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (yashwin) [reply client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: I think it's ok as we haven't released this product. Or we could also add a `// FIXME` comment so we don't miss it.
I will leave this up to you to decide 🙂

File: client/a8c-for-agencies/components/pressable-logo/index.tsx
Comment (yashwin) [inline client/a8c-for-agencies/components/pressable-logo/index.tsx]: [Nit-pick] Do we need these id(s) here?

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (andrii-lysenko) [reply client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: Sounds good. Thanks!

File: client/a8c-for-agencies/components/pressable-logo/index.tsx
Comment (andrii-lysenko) [reply client/a8c-for-agencies/components/pressable-logo/index.tsx]: Thanks! Good catch!

File: client/a8c-for-agencies/components/pressable-logo/index.tsx
Comment (andrii-lysenko) [reply client/a8c-for-agencies/components/pressable-logo/index.tsx:1]: Thanks! Updated.

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (yashwin) [inline client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: [Non-blocker] I suggest we keep this in `/client/a8c-for-agencies/components/sidebar-menu/lib/constants.ts` since it will used in a lot of places.

**Review by yashwin — APPROVED** [review]
Thanks for addressing my comments. 

LGTM now 👍

File: client/a8c-for-agencies/sections/overview/body/hosting/index.tsx
Comment (andrii-lysenko) [reply client/a8c-for-agencies/sections/overview/body/hosting/index.tsx]: Oh, yes, sorry, forgot about this one.

**Pull request merged**
